torch/ao/sparsity/scheduler/lambda_scheduler.py

```diff
@@ -0,0 +1,39 @@
+from .base_scheduler import BaseScheduler
+
+class LambdaSL(BaseScheduler):
```

Comment:
Another scheduler that would be very useful to implement is a delayedScheduler. This scheduler will keep sparsity at 0% till a certain number of epochs and then turn it on based on a lambda function, in effect the lambda fn would take two args, epoch, epoch_start and return a sparsity level of 0 if epoch < epoch_start